storybook/src/stories/Forms/VsiForm.stories.js

```diff
@@ -228,6 +331,18 @@ const formProps = {
   apiEndpointInstanceProfiles: "/mock/api/flavors",
   invalidCallback: invalidCallback,
   invalidTextCallback: invalidTextCallback,
+  composedNameCallback: composedNameCallback,
+  propsMatchState: () => {},
+  invalidVsiVolumeCallback: invalidCallback,
+  invalidVsiVolumeTextCallback: invalidTextCallback,
+  invalidIopsCallback: invalidIopsCallback,
+  invalidIopsTextCallback: invalidIopsTextCallback,
+  vsiVolumeProps: {
+    onSave: () => {},
+    onDelete: () => {},
+    onSubmit: () => {},
+    disableSave: () => {},
```

Comment:
you should add a list of encryption keys to pass in here as well, as the `encryptionKeys` prop is required by the vsi volume form :)